### ISSUE TITLE
Full sized icons and no badge

### ISSUE THREAD

**Issue by dysphasi** [cylonid/NativeAlphaForAndroid#86]
Thank you for all your efforts!

The home screen icon created is a small square set against a white background.

Additionally there is always a native alpha icon badge to the bottom right of the icon.

Is there a way to have a full sized icon images and no badge displayed?

**Comment by cylonid** [comment]
Hi,
these are two different topics, let me address them separately:

- Icon with white background:
Yes, I know that quite a few launchers display icons this way. These launchers expect apps to produce shortcut icons following the "[Adaptive Icon](https://developer.android.com/develop/ui/views/launch/icon_design_adaptive)" pattern. In general this is a good idea - however, Native Alpha cannot use this approach without editing the PNG icon the website is providing and that's not something I looked into yet (and won't in the near future). Currently there is no established standard for website to provide icons in a way it would work as adaptive icon.

TL;DR: Use another launcher, I can recommend open-source Lawnchair but there are many others.


- App badge:
This is another topic which the app has no control over, it is decided by the launcher if shortcuts show their parent app's icon in the right lower corner. A Native Alpha user [on Reddit suggested "Smart Launcher" ](https://www.reddit.com/r/fossdroid/comments/v3v434/comment/io6enqb/?utm_source=share&utm_medium=web2x&context=3) as alternative which does not. I don't know any other one. Afaik Google recommends this badge as security feature, so apps cannot easily produce shortcuts in order to pretend to be other well-known apps, e.g., pretending to be your banking app.

TL;DR: Use another launcher.
